client/app/components/preprint-form-authors/component.js

```diff
@@ -37,9 +37,11 @@ export default Ember.Component.extend(NodeActionsMixin, {
         })
         return contribs;
     }),
-
-
-    valid: Ember.computed.alias('newContributorId'),
+    valid: Ember.observer('contributors.length', function() {
+        let hasContributors = this.get('contributors.length');
+        this.set('isSectionValid', hasContributors);
+        return hasContributors;
+    }),
```

Comment:
This property is an observer because it is not being used in the template (computed properties that are not used in the template will not update). 

However, I just realized this isn't the right property to use for the save icon, which shows whether or not the form changes have been saved. The contributors are automatically saved to the node, so I'm not sure if the icon will ever indicate unsaved changes.